.changeset/beige-tools-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'@solid-design-system/components': patch
+---
+
+Adapt `sd-accordion` and expandable `sd-quickfact` HTML to use `<details />` and `<summary />` elements.
```

Comment:
I personally would even say that this is a feature as this enhances UX a lot (e. g. searchability with the browser search tool so that accordions automatically open.)